apps/web/pages/apps/installed.tsx

```diff
@@ -242,8 +242,9 @@ function IntegrationsContainer() {
                     isGlobal={item.isGlobal}
                     installed={item.installed}
                   />
-                }
-              />
+                }>
+                <AppConfiguration type={item.type} credentialIds={item.credentialIds} />
```

Comment:
This will allow us to injects App configurations dynamically without having to use app specific code in core.